pkg/controller/master/schedulevmbackup/vmbackup.go

```diff
@@ -30,6 +30,11 @@ func (h *svmbackupHandler) OnVMBackupChange(_ string, vmBackup *harvesterv1.Virt
 		h.svmbackupController.Enqueue(svmbackup.Namespace, svmbackup.Name)
 	}
 
+	_, err := checkLHBackupUnexpectedProcessing(h, svmbackup, vmBackup)
```

Comment:
Is the first return value useful? some paths return true.

Reply:
Updated per suggestion, thanks.